src/SaasBundle/Tests/Security/Voter/SubscriptionVoterTest.php

```
@@ -216,6 +219,56 @@ public function testDeniesUnhandledSubscriptionStatus(string $attribute): void
+            subscription: $this->createSubscription(SubscriptionStatus::ACTIVE),
+            featureGate: new NoopFeatureGate(),
+        );
+
+        self::assertVoteResult(VoterInterface::ACCESS_GRANTED, $voter, $attribute);
+    }
+
+    private function featureGateDenying(string $disabledKey): FeatureGate
+    {
+        $featureGate = M::mock(FeatureGate::class);
+        $featureGate->shouldReceive('isEnabled')
+            ->withArgs(static fn (string $key, ?object $for): bool => $key === $disabledKey)
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$for'. (undefined)

(UnusedFormalParameter)